### DISCUSSION TITLE
What steps to follow after saving the annotated data in docbin

### DISCUSSION THREAD

**Discussion by ankitladva11** [tecoholic/ner-annotator#93] [Q&A]
After saving the anotated json data into docbin object , how to use it further in this https://spacy.io/usage/training#training-data, documentation ,havent specificed the steps clearly would like to know about how to proceed ,
Thank you in advance !!

**Comment by alvi-khan** [comment]
If you already have a docbin object (as a .spacy file) you should be able to load it using the instructions from here: [https://spacy.io/api/docbin#from_disk](https://spacy.io/api/docbin#from_disk).

```
doc_bin = DocBin().from_disk("./data.spacy")
```

Although we are happy to help, we will most likely only be able to provide limited support since your question is related to how spaCy works. It might be best to head over to the [spaCy discussion page](https://github.com/explosion/spaCy/discussions) or [Stack Overflow](https://stackoverflow.com/questions/tagged/spacy).
